docs/README.md

```diff
@@ -598,6 +598,12 @@ Check out, builds a maven project and docker images, generating a new alpha vers
           git-username: ${{ secrets.BOT_GITHUB_USERNAME }}
 ```
 
+#### Preview option for maven-build-and-tag
+
+There is a possibility to publish snapshot maven artifacts and docker images from an open PR. 
+In order to use it specify `preview-label` input (or use default `preview`). Create a PR with the `preview-label` label.
+The created artifacts will be tagged as `0.0.1-$GITHUB_PR_NUMBER-$GITHUB_RUN_NUMBER-SNAPSHOT`.
```

Comment:
I wonder if the run number should be used here: do we want to keep dedicated snapshots per run? If so, maybe using the run id instead would be better (?)
And I'm not sure this versioning pattern applies to deployed docker image tags: can you please document those too?

Reply:
I think we may settle on PR number only to avoid multiple snapshots.
As for the docker images, the patter is the same, I'll add it to README.

Reply:
Done.